layouts/shortcodes/ecosystem/registry/search-form.html

```diff
@@ -88,6 +124,16 @@
         {{ end -}}
       </div>
 
+      <!-- Flags Filter Dropdown -->
+    <button class="btn btn-outline-secondary dropdown-toggle" id="flagsDropdown" type="button" 
+    data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Flags</button>
+    <div class="dropdown-menu" id="flagsFilter">
+    <a value="all" class="dropdown-item">Any Flag</a>
+    {{ range $flag := $uniqueFlags -}}
+    <a value="{{ $flag }}" id="flag-item-{{ $flag }}" class="dropdown-item">{{ humanize $flag }}</a>
```

Comment:
<img width="176" alt="Screenshot 2024-10-08 at 11 31 45" src="https://github.com/user-attachments/assets/e5a1c3e3-b144-4978-a659-cfdea187a220">

In the text visible to the user you will need to keep the space between "first" and "party". A way to solve this is by using "first-party" instead of "firstparty", since the humanize will take care of turning that into a space, see https://gohugo.io/functions/inflect/humanize/